src/hu_stuff.c

```diff
@@ -1168,11 +1168,11 @@ static void HU_widget_build_monsec(void)
     }
   }
 
-  kills_color = (kills - extrakills >= totalkills) ? '0'+CR_BLUE : '0'+CR_GOLD;
-  kills_percent_color = (kills >= totalkills) ? '0'+CR_BLUE : '0'+CR_GOLD;
+  kills_color = (kills - extrakills >= totalkills) ? '0'+CR_BLUE : '0'+CR_GRAY;
```

Comment:
I couldn't stand the all-yellow font anymore. What do you say @rfomin ?

Reply:
I assume it's for consistency with vanilla:
![image](https://user-images.githubusercontent.com/5077629/216756373-d52f7d59-d14e-4a82-bba3-19700d549205.png)
Although, I have no preference for font colors.